index.js

```diff
@@ -19,8 +19,29 @@ function fastifyNext (fastify, options, next) {
     .then(() => {
       fastify
         .decorate('next', route.bind(fastify))
+        .decorateReply('nextRender', null)
+        .addHook('onRequest', (request, reply, done) => {
+          reply.nextRender = path => {
```

Comment:
avoid allocating a new closure here and use a function defined in the outer scope.

Reply:
I _think_ I did this in [`ac037d5` (#201)](https://github.com/fastify/fastify-nextjs/pull/201/commits/ac037d52d33a52b2ab23e036618d5cf9dab24b8f)